### ISSUE TITLE
Debugging does not activate vsCode when breakpoint is hit.

### ISSUE THREAD

**Issue by nhdoublek** [microsoft/vscode#138083]
When debugging an angular application using (vsCode 1.62.3) with the Chrome debugger and a break point is hit when the Chrome browser is the active application, then switching to vsCode by clicking on the vsCode task bar item does not activate vsCode and display the breakpoint that was hit.  The work around is to toggle vsCode as the active application and then it will update and run to the break point. It works fine in version 1.58.2.  We currently cannot upgrade to a newer version of vsCode because this issue is too annoying.  Please correct this issue so we may upgrade.  P.S. This issue has happened previously and was corrected in a prior release.  Perhaps it could be added to automated QA testing so it does not get shipped as broken in the future.

**Comment by devin87** [comment]
I also encountered this problem in vscode 1.62.3, it works fine in version 1.58.2. I accessed the Node.js api in my browser to trigger a breakpoint, and then returned to the vscode window, but the breakpoint was not highlighted, and debugging could not proceed to the next step. I had to switch to another process (such as the browser). ), and then return to the vscode window to debug the function properly. This problem has a great impact on debugging, and hope to fix it soon.
